packages/cli/src/oclif/commands/integrations.js

```diff
@@ -3,11 +3,16 @@ const { buildFlags } = require('../buildFlags');
 
 const { listApps } = require('../../utils/api');
 
-class AppsCommand extends BaseCommand {
+class Integrations extends BaseCommand {
```

Comment:
Shouldn't we name it `IntegrationsCommand`?

Reply:
yep, copy pasting 😁 doesn't really matter, but it's probably good. We could also name them all `Command`, since it's renamed at import